spec/lib/blog_fetcher_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe BlogFetcher do
+ describe ".feed" do
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.